MiniApp/Classes/Protocols/MiniAppUserInfoDelegate.swift

```diff
@@ -18,4 +21,18 @@ public extension MiniAppUserInfoDelegate {
     func getProfilePhoto() -> String? {
         return nil
     }
+
+    func getAccessToken(miniAppId: String, completionHandler: @escaping (Result<MATokenInfo, MASDKCustomPermissionError>) -> Void) {
+        completionHandler(.failure(.failedToConformToProtocol))
+    }
+}
+
+public class MATokenInfo: Codable {
+    let token: String
+    let validUntil: Date
+
+    public init(accessToken: String, expirationDate: Date) {
+        self.token = accessToken
+        self.validUntil = expirationDate
```

Comment:
Regarding my above comment, basically here could you set `self.validUntil = expirationDate.timeIntervalSince1970`?

Reply:
expirationDate.timeIntervalSince1970 returns a Double. I have updated the code to return Number `Int` now.